src/Components/Countries.jsx

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable no-mixed-spaces-and-tabs */
+/* eslint-disable no-tabs */
+/* eslint-disable no-unused-vars */
+/* eslint-disable max-len */
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Kindly remove every eslint disable rule and instead fix the issue in the code.